{{cookiecutter.project_slug}}/.github/workflows/workflow_linting.yml

```
@@ -0,0 +1,27 @@
+name: Snakemake linting check
+
+on:
+  push:
+  workflow_call:
+
+env:
+  POETRY_VERSION: 2.1.0
+
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install micromamba
+        uses: mamba-org/setup-micromamba@v2
+        with:
+          environment-file: environment.yml
+          init-shell: >-
+              bash
```

I think this can be a oneliner
```suggestion
          init-shell: bash
```